COVERAGE.md

```diff
@@ -0,0 +1,42 @@
+# Coverage reports
+
+## Caveat
+
+Unfortunately, until we are capable of using Node.js's builtin ESM support as
+well as implementing source maps in all of our source-to-source transforms (such
+as `@agoric/bundle-source`, `@agoric/transform-metering`, and
+`@agoric/static-module-record`), the coverage line numbers will be out-of-sync
+with reality.
```

Comment:
That's a pretty big caveat. Do the percentages still seem to be accurate? Is there a human-manageable way to get from the line numbers this emits to the actual source code? Or should we preface all the reports with a note that we should ignore the line numbers entirely and just look at the percentages?

Reply:
Also, does this affect basic unit tests that use `-r esm` (like everything we do) but do not use `bundle-source` or `import-bundle`? I'd be happy with getting accurate coverage for library code that we can test outside the context of swingset.

Reply:
So the basic plan is: if we adopt the Node.js `esm` package patch I made in this PR, we can do a gradual bottom-up migration of our packages to be dual `-r esm` and Node.js ESM (`nesm`) compatible.

I've changed the HTML file generation only to coverage-test the packages which can support nesm and explicitly implement a `test:c8` script.  Currently only `swing-store-simple` implements that (and incidentally, also supports `resm`).